Imagens do que foi gerado até agora

tfcorrea opened the pull request:
Link para o zeplin
https://zpl.io/a7KDlAW

tfcorrea committed:
Imagens do que foi gerado até agora

Link para o zeplin
https://zpl.io/a7KDlAW

samuelgrigolato committed:
Move mockup images to a new folder

Also enhance the README.md file.

samuelgrigolato commented:
@tfcorrea mais uma vez, parabéns pelos mockups! Ficaram muito bons rsrs, se o projeto continuar espero poder contar com mais deles :D. Dei apenas um tapa no local onde os versionou e aproveitei para melhorar o README.md também. Valeu!

samuelgrigolato committed:
Workaround pytest-flask bug

Some library is breaking pytest-flask's response.json cached attribute.
In order to fix that I've changed the test code to parse the JSON
directly from the field response.data.
